platform/landing-page/src/test/java/org/codice/ddf/landing/LandingPageTest.java

```diff
@@ -70,6 +74,9 @@ public static void setupLandingPage() throws IOException {
         Arrays.asList(secondDateNoLeadingZeroes, noDate, firstDateLeadingZeroes);
     sorted = Arrays.asList(firstDateLeadingZeroes, secondDateNoLeadingZeroes, noDate);
     landingPage.setAnnouncements(unsorted);
+    keywordRegistry = new KeywordRegistryImpl();
+    keywordRegistry.setKeywordList(ImmutableList.of("Data Source=Data Source"));
```

Comment:
Do we have a test that verifies the replacement? Should we set the test with "Data Source=Sources" or something to verify it works?

Reply:
There isn't a way to check that the replacements work. However, I could add additional assertions to check that the default mappings are correct